scripts/hooks/nucypher_dkg.py

```diff
@@ -1,21 +1,9 @@
-# Run nucypher dkg on 'lynx'/'mumbai' by running:
-#
-#    python scripts/hooks/nucypher_dkg.py <ETH_PROVIDER_URI> <COORDINATOR_PROVIDER_URI> <RITUAL_ID> <SIGNER_URI_FOR_COORDINATOR_NETWORK>
-#
-# For example:
-#
-# Use existing ritual id (eg. id '0'):
-#    python ./scripts/hooks/nucypher_dkg.py <ETH_PROVIDER_URI> <COORDINATOR_PROVIDER_URI> 0
-#
-# Go through entire process of initiating a ritual, waiting for it to finish, then use for encryption/decryption:
-#    python ./scripts/hooks/nucypher_dkg.py <ETH_PROVIDER_URI> <COORDINATOR_PROVIDER_URI> -1 <SIGNER_URI_FOR_COORDINATOR_NETWORK>
-#
-
 import random
 import sys
 import time
 
 import click
+import maya
```

Comment:
Just out of curiosity: What are the advantages of using maya instead of canonical time here?

Reply:
`maya` makes displaying the elapsed time in seconds easier - https://github.com/derekpierre/nucypher/blob/clickerize-script/scripts/hooks/nucypher_dkg.py#L176.

`time.time() - start_time` produces a float that needs rounding (eg. `18.45042395591736`) whereas `maya.now() - start_time)` produces a `datetime.timedelta`.

Reply:
Thanks!